### DISCUSSION TITLE
Adding country flags icons

### DISCUSSION THREAD

**Discussion by santigandolfo** [TiendaNube/nimbus-design-system#187] [Ideas]
It would be nice to have icons of country flags natively in the library(specifically flags for the countries we are in).
We are currently using downloaded svgs from https://flagicons.lipis.dev/, but would love to be able to use them directly from Nimbus Icons.

**Comment by jcampos89** [comment]
@santigandolfo thank you for your contribution!
In internal discussions, we have concluded that flags are not compatible with the icon language we use.
Our suggestion is to use flag emojis, as has been done before on the partners portal.
We're happy to answer any other questions you have.
